src/main/java/com/github/hey_world_team/currency_converter/model/BestConversion.java

```diff
@@ -0,0 +1,15 @@
+package com.github.hey_world_team.currency_converter.model;
+
+import lombok.*;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class BestConversion {
+    private BigDecimal convertedAmount;
```

Comment:
what is mean this value?

Reply:
convertedAmount- the best rate you can get